minidump-stackwalk/src/main.rs

```diff
@@ -313,7 +312,6 @@ native debuginfo formats. We recommend using a version of dump_syms to generate
                 symbols_paths,
```

Comment:
You can now just drop the use of `MultiSymbolProvider` entirely here, and just do like:

```rust
let mut provider = if !symbols_urls.is_empty() {
    Box::new(Symbolizer::new(http_symbol_supplier( /* … */ )))
else {
    Box::new(Symbolizer::new(simple_symbol_supplier( /* … */ )))
};
```

Reply:
While this is true, I like keeping the machinery here for future improvements (and to keep it working). The overhead has got to be completely trivial.